examples/client/listeverything/main.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2025 The Go MCP SDK Authors. All rights reserved.
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file.
+
+// The listeverything command lists all features of a stdio MCP server.
```

Comment:
Since it only lists features, maybe listfeatures? or mcp-features or mcp-inspect?